.github/workflows/ci.yml

```diff
@@ -53,7 +47,9 @@ jobs:
 
           - operating-system: 'ubuntu-20.04'
             php-version: '8.0'
-            composer-flags: '--ignore-platform-req=php' # as this is a version not yet officially supported by PHP CS Fixer
```

Comment:
i wonder, maybe we should add 8.1 ?

Reply:
sounds like a good idea... for separate PR ;)